pages/docs/doc/igpm.json

```
@@ -0,0 +1,218 @@
+{
+  "tags": [
+    {
+      "name": "INDICES",
+      "description": "Informações referentes aos indices \nIGP-M"
+    }
+  ],
+  "paths": {
+    "/indices/igpm/v1": {
+      "get": {
+        "tags": ["INDICES"],
+        "summary": "IGPM - Lista de indices",
+        "description": "Retorna todos os valores de IGP-M desde sua criação",
+        "responses": {
+          "200": {
+            "description": "Success",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/components/schemas/IGPM_LISTA"
+                  }
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "$ref": "#/components/schemas/IGPM_INTERNAL_ERROR"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/indices/igpm/v1/ultimos": {
+      "get": {
+        "tags": ["INDICES"],
+        "summary": "IGPM - Últimos indices",
+        "description": "Retorna todos os ultimos últimos valores do IGP-M de acordo com a quantidade informada",
+        "parameters": [
+          {
+            "name": "limit",
+            "in": "query",
+            "description": "Quantidade de registros a serem obtidos",
```

Seria quantidade de meses ou registros? Só pro caso de haver mais de um índice pro mesmo mês
```suggestion
            "description": "Quantidade de meses a serem obtidos",
```